apisix/discovery/dns/schema.lua

```diff
@@ -24,6 +24,9 @@ return {
                 type = "string",
             },
         },
+        resolvconf = {
```

Comment:
We should add this to config-default.yaml

Reply:
config-default already has `servers` https://github.com/apache/apisix/blob/01763b5ddae1ba4f6d52778fad420aa4221c3c44/conf/config-default.yaml#L279 
and only one of `servers` or `resolvconf` is needed. Should I add it anyways?

Reply:
You should add the `resolv_conf` to the config-default.yaml, or nobody will find this attribute

Reply:
Okay sounds good. Adding it..